src/test/ldap/t/001_auth.pl

```diff
@@ -376,9 +368,7 @@ sub test_access
 $node->append_conf('pg_hba.conf',
 	qq{local all all ldap ldapurl="$ldaps_url/$ldap_basedn??sub?(uid=\$username)" ldaptls=1}
 );
-$node->restart;
-
-$ENV{"PGPASSWORD"} = 'secret1';
-test_access($node, 'test1', 2, 'bad combination of LDAPS and StartTLS');
+$node->stop;
+is($node->start(fail_ok => 1), 0, "bad combination of LDAPS and StartTLS");
```

Comment:
Maybe it's worth checking for an error message in the log?
```suggestion
is($node->start(fail_ok => 1), 0, "bad combination of LDAPS and StartTLS");
ok($node->log_contains(
	"cannot use 'ldaptls' with 'ldaps' scheme or 'ldapurl' start with 'ldaps://'"),
	"bad combination of LDAPS and StartTLS");
```

Reply:
applied